src/test/resources/features/Pillar2SubmissionSetUpPage.feature

```diff
@@ -10,18 +10,18 @@ Feature: Pillar2 submission set up page
   @p2zap_p2accessibility
   Scenario: 2 - User logins as organisation user and standard role
     Given Organisation Assistant User logs in to subscribe for Pillar2 Submission
-    Then I should be on P2 Submission Assistant role KB page
+    Then I should be navigated to P2 Submission Assistant role KB page of Pillar2 Submission
 
   @p2zap_p2accessibility
   Scenario: 3 - User logins as Agent
     Given Agent User logs in to subscribe for Pillar2 Submission
-    Then I should be on P2 Submission Agent affinity KB page
+    Then I should be navigated to P2 Submission Agent affinity KB page of Pillar2 Submission
```

Comment:
is this the same step, just worded differently?

Reply:
yes, because this is on pillar2-submission app